content/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages.md

```diff
@@ -125,11 +125,7 @@ To ensure the repository's URL is correct, replace REPOSITORY with the name of t
 {% data variables.product.prodname_registry %} will match the repository based on the URL, instead of based on the package name. If you store the *package.json* file outside the root directory of your repository, you can use the `directory` field to specify the location where {% data variables.product.prodname_registry %} can find the *package.json* files.
 
 ```shell
-"repository" : {
-    "type" : "git",
-    "url": "ssh://git@{% if currentVersion == "free-pro-team@latest" %}github.com{% else %}<em>HOSTNAME</em>{% endif %}/<em>OWNER</em>/<em>REPOSITORY</em>.git",
-    "directory": "packages/name"
```

Comment:
It explains the "directory" value in the previous paragraph

Reply:
Good catch! I don't think we need to document the `directory` value here (it isn't used by GitHub Packages).